### PR TITLE
Run Project Implementation Initial Commits

### LINKED ISSUE #14: Run Project Implementation
The current Run Project button does not redirect users to a page where they can upload and run PowerSynth files. To resolve this issue, the button needs to be updated to take users to a dedicated page for file uploads.

**Expected Result:** The Run Project button should redirect users to a page where they can upload PowerSynth files and run it.

**Sample Designs:** Dr. Peng has shared sample designs on the ssh server (peng-srv2.csce.uark.edu) in the "sample designs" folder.

**SSH Server Access:**
- Host: peng-srv2.csce.uark.edu
- Account: cap22
- Password: [Password available in the team's chat]
- Sample designs location: /nethome/cap22/Sample_Designs

**Acceptance Criteria:**
- The Run Project button should redirect users to a dedicated page for file uploads.
- Users should be able to upload PowerSynth files and submit job to backend.
- The dedicated page should have a clear and user-friendly interface for file uploads and running.

### PR DESCRIPTION
**Pull request by kevin-german**
Initial upload of files to handle implementation for File Upload UI.

## Describe your changes
Created files for Run Project UI
## Issue link and documentation link
[💻Issue](https://github.com/PowerSynth/front-end/issues/14)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added the necessary documentation.

### CODE REVIEW

File: src/components/WelcomePage.css [hunk 1]
```diff
@@ -24,3 +24,5 @@
   cursor: pointer;
   width: 200px;
 }
+
+
```
Comment (connorielliott) [inline src/components/WelcomePage.css]: Possibly undo this change to keep git history clean?

**Review by connorielliott — DISMISSED** [review]
Possibly just address my comment, other then that changes look great and the upload file is very nice looking 👍

**Review by connorielliott — DISMISSED** [review]

**Review by connorielliott — DISMISSED** [review]

**Review by connorielliott — APPROVED** [review]

**Review by atejedag — APPROVED** [review]
Looks good
